spec/controllers/account/tasks_controller_spec.rb

```diff
@@ -3,21 +3,59 @@
 RSpec.describe Account::TasksController, type: :controller do
   render_views
 
-  context 'PUT /:move' do
-    let!(:user) { create(:user) }
-    let!(:workspace) { create(:workspace, user_id: user.id) }
-    let!(:project) { create(:project, workspace_id: workspace.id) }
-    let!(:task) { create_list(:task, 5, project_id: project.id) }
-
-    before do
-      sign_in user
-      user.update(projects: [project])
+  let!(:user) { create(:user) }
+  let!(:workspace) { create(:workspace, user_id: user.id) }
+  let!(:project) { create(:project, workspace_id: workspace.id) }
+  let!(:task) { create_list(:task, 5, project_id: project.id) }
+
+  before do
+    sign_in user
+    user.update(projects: [project])
+  end
+
+  after do
+    sign_out user
+  end
+
+  context 'GET /tasks/new' do
+    it 'should create a new task' do
+      get :new, params: {
+        project_id: project.id,
+      }
+      expect(response).to be_successful
+    end
+  end
+
+  context 'POST /tasks' do
+    it 'should create a task and redirect to project page' do
+
+      post :create, params: {
+        project_id: project.id,
+        task: {
+          title: 'Test task',
+          description: 'test description'
+        }
+      }
+      expect(response).to redirect_to account_workspace_project_path(workspace.id, project.id)
     end
 
-    after do
-      sign_out user
+    it 'shouldn`t create a task and render page new' do
+      get :new, params: {
```

Comment:
something wrong here. You shouldn't make 2 requests in 1 test

Reply:
controller tests should be based on a contextual request (REST as basic + additional)
```
describe 'GET #new' do
end

describe 'POST #create' do
end

describe 'PATCH #complete' do
end

#...
```